### ISSUE TITLE
Files written in DOS format

### ISSUE THREAD

**Issue by SlimeDog** [Multiverse/Multiverse-Inventories#437]
### Information

* **OS:** CentOS Linux release 7.9.2009 (Core)
**Java:**
```
java version "11.0.10" 2021-01-19 LTS
Java(TM) SE Runtime Environment 18.9 (build 11.0.10+8-LTS-162)
Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11.0.10+8-LTS-162, mixed mode)
```

* **Server version:** 
```
>version
[22:50:46] [Server thread/INFO]: This server is running CraftBukkit version 3018-Spigot-db99f82-a021e33 (MC: 1.16.5) (Implementing API version 1.16.5-R0.1-SNAPSHOT)
[22:50:46] [Server thread/INFO]: You are running the latest version
```

* **Full output of `/mv version -p`:** https://j.mp/3buDRGh

* **Server log:** irrelevant

### Help request

**Problem**
plugins/Multiverse-Inventories/config.yml and plugins/Multiverse-Inventories/groups.yml are rewritten on every server start.
- Why?
- Why are they written in DOS format, rather than the native file format of the OS?

**What I have tried**
- Stop server
- dos2unix plugins/Multiverse-Inventories/config.yml plugins/Multiverse-Inventories/groups.yml
- Start server
- Verify that files have been rewritten in DOS format
```
plugins/Multiverse-Inventories/config.yml:                  ASCII text, with CRLF, LF line terminators
plugins/Multiverse-Inventories/groups.yml:                  ASCII text, with CRLF, LF line terminators
```

**Comment by nicegamer7** [comment]
I believe all MV config files are rewritten on startup. The issue with CRLF line endings is exclusive to MV Inventories though, and is fixed with #361.

**Comment by SlimeDog** [comment]
> all MV config files are rewritten on startup

True. The questions was why, given that the substantive content does not change (for me; I track all changes and make them manually when updating the plugins)?

> The issue with CRLF line endings is exclusive to MV Inventories though, and is fixed with #361.

Great, thanks. Patiently awaiting a build. :)

**Comment by nicegamer7** [comment]
As the why this happens, I'm not sure. I guess that can be looked into.
